jobs/shipwell_backend_engineer.html

```diff
@@ -0,0 +1,82 @@
+---
+title: Backend Engineer - Python - Austin
+company: Shipwell
+url: https://www.shipwell.com
+location: Austin, TX
+contract: permanent
+contact:
+    name: Vahid Behzadi
+    email: vahid@shipwell.com
+    Apply here: https://shipwell.bamboohr.com/jobs/view.php?id=27
+created: !!timestamp 2018-04-25
+tags:
+  - python
+  - aws
+  - django
+  - Startup
+  - hiring
+  - austin
+  - texas
+  - capital factory
+  - first round capital
+  - salary
+  - equity    
+---
+
+# Title
+Backend Engineer - Full time – Austin, TX
+
+## Sub-title
+The mission of Shipwell is to build a connected freight platform that is the backbone of how shippers and freight movers connect, communicate, transact, track and derive analysis of their businesses and supply chains. We are a fast growing company and have assembled a team that is passionate about what we do and regularly exceed customer's expectations in both the experience we deliver and the service we provide. 
+
+We are looking for strong contributors with a history of delivering high-quality, scalable applications to join the backend engineering team. As a Backend Engineer, you will be part of a team that develops the APIs and services powering the platform this includes things like quoting shipments, tracking, searching, notifications and messaging to name a few. You will help design performant, scalable applications and will work hand in hand with Front-End and UX Engineers in ensuring that experiences work seamlessly, end to end. You will maintain and extend existing systems and will have lots of opportunities to work on new ones and to push the envelope of what we do as a team.
+
+**Day to Day Responsibilities:**
+
+Own the end to end development of features which includes clarifying requirements, designing solutions, writing code and ensuring that features are well tested
+
+Write high quality, clean and efficient code
+
+Ensure that design is architecturally sound and follows engineering best practices with regards to scalability, maintainability, and reuse
+
+Provide timely support to both internal and external users of the system
+
+**Essential Skills:**
+
+BS/MS/ Ph.D. in Computer Science or related field
```

Comment:
So basically, any degree? But do you really need a degree if you are open to somebody from non CS backgrounds? It might be worth looking for people with a practical can-do attitude.

Reply:
this is pretty close to a hard requirement. Trust me, I know there are tons of great engineers without degrees in CS. But for now, we're on that path. That could change in the future.